README.md

```diff
@@ -1,12 +1,17 @@
-### Running tests:
-
-To run the tests, use `nosetests`
+### Instructions on running tests:
```

Comment:
The phrase "Instructions on" is redundant. See https://github.com/nyu-devops/lab-flask-tdd

Reply:
No problem. Feel free to change it back @jh7494